### ISSUE TITLE
[Accessibility]: On 200% zoom, some elements in the screen are not visible\truncated.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7254]
**Test Environment:**
Download [Eclipse IDE](https://www.eclipse.org/downloads/download.php?file=/oomph/epp/2022-06/R/eclipse-inst-jre-win64.exe&mirror_id=1248)
OS: Windows_11 Version: 22H2 (OS Build: 22598.200)

**Pre-Requisites (if any):**
1. Download latest version of Eclipse 
2. Download “Azure Toolkit for Eclipse” from “Help” window (Navigation: Open Eclipse -> Help -> Eclipse Marketplace). 
3. Change the system display resolution to 1280*1024. 
4. Change the display scaling settings to 400%. 

**Repro Steps:**
1. Download and Install Eclipse([Eclipse IDE](https://www.eclipse.org/downloads/download.php?file=/oomph/epp/2022-06/R/eclipse-inst-jre-win64.exe&mirror_id=1248)) 
2. Tab till tools-->Azure-->Azure Sign in and press enter 
3. Set the display resolution to 1280*768 and zoom upto 200%. 
4. Verify that all the elements of "Eclipse Azure Sign In" section are accessible and MAS Compliant.

**Actual Result:**
**On applying 200%, text present under 3rd radio button i.e., Device Radio button is not visible. Also, 4th radio button and the text present under the button is not visible. 

Similar issue can be seen in the following. 
Deploying web app to Azure_Deploy Web App scenario---->Open Java EE - Eclipse application-->Install 'Azure Toolkit for Eclipse' from Marketplace.-->Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription-->Tab till File-->New-->Project--->Dynamic Web Project-->Tab till Next button and press enter--> In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App-->Opens "Deploy Web App" dialog box.-->Verify the "Select App Service to deploy to:" element is MAS compliant. 
Deploying web app to Azure_Create New Web App scenario---->Open Java EE - Eclipse application-->Install 'Azure Toolkit for Eclipse' from Marketplace.-->Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription-->Tab till File-->New-->Project--->Dynamic Web Project-->Tab till Next button and press enter--> In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App-->Opens "Deploy Web App" dialog box.-->Enter on "Create" button-->Verify the "Create Azure App Service" is MAS compliant. **

**Expected Result:**
Every element should be visible after applying 200% zoom. 

Attachments:
![1955090_A11y_Eclipse_Azure Sign in_Resize](https://user-images.githubusercontent.com/93735775/191558593-8674c137-e0bd-4df3-966b-826552015d01.jpg)
![merge_from_ofoct (1)](https://user-images.githubusercontent.com/93735775/191558823-f04206f3-b145-4115-9a73-648c323b5814.jpg)
![merge_from_ofoct](https://user-images.githubusercontent.com/93735775/191558846-73d52653-933a-4335-8fe2-dd21784b48c5.jpg)

**Comment by mportuga** [comment]
We are not responsible for azure, that is managed by someone else
